lib/liquid/extensions.rb

```diff
@@ -25,6 +26,12 @@ def to_liquid
   end
 end
 
+class BigDecimal
+  def to_liquid
+    to_f
```

Comment:
I guess I don't really see the problem with `Utils.to_number(input).abs` since that keeps it as a BigDecimal.

There are some sketch conversions like one from BigDecimal to a Float in the round filter and Utils.to_number converting a Float to BigDecimal.  I'm not sure what is the best way to handle that in a backwards manner though, since the default behaviour for BigDecimal#to_s is weird.